core-server/src/main/java/org/orbisgis/orbisserver/control/xml/GetCapabilitiesController.java

```diff
@@ -0,0 +1,167 @@
+/**
+ * OrbisServer is part of the platform OrbisGIS
+ *
+ * OrbisGIS is a java GIS application dedicated to research in GIScience.
+ * OrbisGIS is developed by the GIS group of the DECIDE team of the
+ * Lab-STICC CNRS laboratory, see <http://www.lab-sticc.fr/>.
+ *
+ * The GIS group of the DECIDE team is located at :
+ *
+ * Laboratoire Lab-STICC – CNRS UMR 6285
+ * Equipe DECIDE
+ * UNIVERSITÉ DE BRETAGNE-SUD
+ * Institut Universitaire de Technologie de Vannes
+ * 8, Rue Montaigne - BP 561 56017 Vannes Cedex
+ *
+ * OrbisServer is distributed under LGPL 3 license.
+ *
+ * Copyright (C) 2015-2017 CNRS (Lab-STICC UMR CNRS 6285)
+ *
+ * This file is part of OrbisGIS.
+ *
+ * OrbisServer is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * OrbisServer is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * OrbisServer. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, please consult: <http://www.orbisgis.org/>
+ * or contact directly:
+ * info_at_ orbisgis.org
+ */
+
+package org.orbisgis.orbisserver.control.xml;
+
+ import net.opengis.ows._2.*;
+ import org.orbiswps.server.WpsServer;
+ import org.orbiswps.server.model.JaxbContainer;
+ import org.wisdom.api.DefaultController;
+ import org.wisdom.api.annotations.Controller;
+ import org.wisdom.api.annotations.Parameter;
+ import org.wisdom.api.annotations.Route;
+ import org.wisdom.api.annotations.View;
+ import org.wisdom.api.exceptions.HttpException;
+ import org.wisdom.api.http.HttpMethod;
+ import org.wisdom.api.http.Result;
+ import org.wisdom.api.templates.Template;
+ import org.slf4j.Logger;
+ import org.slf4j.LoggerFactory;
+ import org.xnap.commons.i18n.*;
+
+ import org.orbisgis.orbisserver.control.web.IndexController;
+
+ import javax.xml.bind.JAXBElement;
+ import javax.xml.bind.JAXBException;
+ import javax.xml.bind.Marshaller;
+ import javax.xml.bind.Unmarshaller;
+
+ /**
+  * Instance of DefaultController containing a WpsServer.
+  *
+  * @author Guillaume MANDE
+  */
+ @Controller
+ public class GetCapabilitiesController extends DefaultController {
+
+  /**
+    * IndexController's object, used to get some useful attributes.
+    */
+   private IndexController indexController = new IndexController();
```

Comment:
You should not instantiate an `IndexController` inside the `GetCapabilitiesController` if it is only to get the `WpsCapabilitiesType` object. If you need to manage the access to the wps information and to access them from different classes, maybe the better thing to do is to have one class which return the well formatted information (xml or process list).